src/TGPPL-1.0.xml

```diff
@@ -0,0 +1,264 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="TGPPL-1.0" name="Transitive
+   Grace Period Public Licence 1.0" listVersionAdded="3.23">

```

Comment:
The space in XML content is insignificant. But in values in attributes, it is significant. E.g. see https://learn.microsoft.com/en-us/dotnet/visual-basic/programming-guide/language-features/xml/white-space-in-xml-literals

So this attribute is `name="Transitive    Grace Period Public Licence 1.0"`.
You can change it to:
```
   <license isOsiApproved="false" licenseId="TGPPL-1.0"
     name="Transitive Grace Period Public Licence 1.0" listVersionAdded="3.23">
```
and it will be ok.

Reply:
thanks! fixed